pkg/sql/logictest/testdata/logic_test/crdb_internal_tenant

```diff
@@ -0,0 +1,12 @@
+# LogicTest: 3node-tenant
+
+query II
+SELECT count(distinct(node_id)), count(*)  FROM crdb_internal.node_runtime_info
```

Comment:
It'd be good to have at least one test case for all the internal tables and functions, to ensure they don't regress. I'd copy the entire `crdb_internal` suite, and then modify it for tenants, so that our testing is at least as good as that.

Reply:
Good point. Doing this in #55860 and will backport both these commits to 20.2.1